### DISCUSSION TITLE
Dockerized Version of UERANSIM

### DISCUSSION THREAD

**Discussion by Spectranis** [aligungr/UERANSIM#299] [General]
Hello,

for tests with multiple simulated gNB's I've created a [dockerized Version of UERANSIM](https://github.com/COCUSAG/UERANSIM_docker)
So far I've used it to create up to 30 gNB instances to test open5gs .

Regards
Marcel

**Comment by aligungr** [comment]
Thank you Marcel, I appreciate your work. I put a link to the repository in the UERANSIM Wiki page.
